flake.nix

```diff
@@ -6,30 +6,26 @@
   description = "automation-service";
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs";
+    # for PureScript 0.15.11
+    nixpkgs.url = "nixpkgs/9957cd48326fe8dbd52fdc50dd2502307f188b0d";
     flake-utils.url = "github:numtide/flake-utils";
-    easy-purescript-nix.url = "github:justinwoo/easy-purescript-nix";
   };
 
-  outputs = { self, nixpkgs, flake-utils, easy-purescript-nix }:
+  outputs = { self, nixpkgs, flake-utils }:
     flake-utils.lib.eachSystem ["x86_64-linux"] (system:
       with nixpkgs.legacyPackages.${system};
       let
         t = lib.trivial;
         hl = haskell.lib;
         haskellPackages = haskell.packages.ghc927;
-        easy-ps = easy-purescript-nix.packages.${system};
 
         name = "automation-service";
 
         project = devTools:
           let
             addBuildTools = (t.flip hl.addBuildTools) (devTools ++ [
-              easy-ps.purs-0_15_10
-              easy-ps.spago
-              easy-ps.purescript-language-server
-              easy-ps.purs-tidy
-              pkgs.nodejs_20
+              pkgs.purescript
+              pkgs.nodejs_18
```

Comment:
Addresses an issue with Global Registrator with Happy Dom, although I'm leveraging the fact that whatever is at 18 in nix is lower than 18.18.2, which is apparently >= is problematic